Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM debian:testing
+RUN apt-get update && apt-get -y install git python-pip python-dev
```

Comment:
przeczytałem tę linię jeszcze raz i jak nie będziemy zaciągać z githuba, to `git` jest do usunięcia z installa.